### ISSUE TITLE
why set axis=1 in batch normalization layer?

### ISSUE THREAD

**Issue by lc123623** [4thgen/DCGAN-CIFAR10#1]
we usually set axis=-1 in bn layer to calculate the mean and the std of a every channel, but here you set axis=1.  In fact I have trained your model, when I set axis=-1, the result is worse then when axis=1. So I want to know why?

**Comment by akanimax** [comment]
Isn't it set to -1?
here -> [https://github.com/4thgen/DCGAN-CIFAR10/blob/master/ops.py#L19](https://github.com/4thgen/DCGAN-CIFAR10/blob/master/ops.py#L19)

The batchnormalization axis is set to -1 because, that's the channel axis.
